GovUk.Frontend.ExampleApp/Views/TextInput/Index.cshtml

```diff
@@ -12,6 +13,7 @@
 
 <form asp-controller="TextInput" asp-action="Post" method="post" novalidate>
     <span class="govuk-caption-xl">Example application</span>
+        <div class="govuk-error-prefix" data-govuk-error-prefix=@Localizer["Error:"]></div>
```

Comment:
You don't need to insert a new element for this data. You could put the attribute on any element. In this view for form element would be a good place to put it.

Also `Localizer` is a variable so should by convention be lowercase.